generators/python-v2/ast/src/__test__/__snapshots__/Class.test.ts.snap

```diff
@@ -13,19 +13,30 @@ class MyDataClass:
 "
 `;
 
-exports[`class > class with de-indented multi-line string 1`] = `
-"class MyClass:
-    multiline_string: str = """Hello
-World"""
-"
-`;
-
 exports[`class > class with generic parent reference 1`] = `
 "class MyClass(MyParentClass[MyParentType]):
     pass
 "
 `;
 
+exports[`class > class with various forms of multi-line strings 1`] = `
+"class MyClass:
+    has_newline_chars__basic: str = """Hello,
+World!"""
+    has_no_newline_chars__basic: str = """Hello, World!"""
+    has_newline_chars__start_on_new_line: str = """\\
+Hello,
+World!"""
```

Comment:
The intention is to only have a single `\` character here. The writer seems to be adding a second `\` for some reason. It's unclear if this is:
1. A fragment of printing out the string and it wouldn't actually manifest when writing the file;
2. A property of the writer where it auto-escapes `\` and needs to be overridden
3. Something else?

Reply:
Hmm.. not sure but when I do the following in the TypeScript AST I can confirm that only a single `\` is written to the file:

```
writer.write("\\");
```